### ISSUE TITLE
The requested module './pyproxy.gen.js' does not provide an export named 'wrapNamespace' in Node

### ISSUE THREAD

**Issue by owenthereal** [pyodide/pyodide#1863]
I got the `The requested module './pyproxy.gen.js' does not provide an export named 'wrapNamespace'` error with pyodide `0.18.1` in node `v16.10.0`:

```js
async function start() {
    let pyodide_pkg = await import("pyodide/pyodide.js");

    let pyodide = await pyodide_pkg.loadPyodide({
        indexURL: <path>,
    });

    await pyodide.runPythonAsync("1+1");
}

start()

```

```
node --experimental-repl-await lib/funtime.js
file:///Users/owen/code/src/github.com/owenthereal/pyodide-test/node_modules/pyodide/pyodide.js:13
import { wrapNamespace } from "./pyproxy.gen.js";
         ^^^^^^^^^^^^^
SyntaxError: The requested module './pyproxy.gen.js' does not provide an export named 'wrapNamespace'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:179:5)
    at async Loader.import (node:internal/modules/esm/loader:178:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:437:15)
    at async start (/Users/owen/code/src/github.com/owenthereal/pyodide-test/lib/funtime.js:2:23)
```

**Comment by hoodmane** [comment]
What does your `pyproxy.gen.js` look like? Mine ends with:
```js
export function wrapNamespace(ns) {
  return new Proxy(ns, NamespaceProxyHandlers);
}
```
lines 1607 -- 1609. Maybe try deleting it and rebuilding Pyodide?

**Comment by owenthereal** [comment]
This is the file: https://gist.github.com/owenthereal/e18ecf840562a756482a29a957d26077. I don't see the `wrapNamespace` function`

**Comment by hoodmane** [comment]
What commit was it generated from?

**Comment by owenthereal** [comment]
I have no idea which commit but it's downloaded from npm. The version is 0.18.1

**Comment by hoodmane** [comment]
> it's downloaded from npm

This presumably means the npm release is messed up =( Thanks for the report!
@rth How to fix?

**Comment by rth** [comment]
Hmm, I probably didn't rerun the C preprocessor as part of the build when released. Hmm, it doesn't look like semantic versionning allows to have post-release tags. So I can push it as 0.18.2 otherwise.

**Comment by owenthereal** [comment]
FYI, I copied the functions in this file: https://github.com/pyodide/pyodide/blob/89682dabfe32cb9f6508a18a5e838e349801675a/src/core/pyproxy.js#L1435-L1480 to hack around the missing `wrapNamespace` function issue.

**Comment by rth** [comment]
Sorry for the delay. It should be fixed in the 0.18.2 release available now on npm. Please comment if it's still an issue.
